test_integration/02_stake_and_mint/utils/assert_confirm_stake_intent.js

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const assert = require('assert');
+
+/**
+ *  Class to assert confirm stake intent.
+ */
+class AssertConfirmStakeIntent {
+    /**
+     *  This verifies event.
+     *
+     * @param event Event object after decoding.
+     * @param stakeRequest Stake request. {amount: *, gasPrice: *,gasLimit:*
```

Comment:
For documenting an object it's better to have a separate `@typedef`, or document the fields with the `stakeRequest.amount` syntax, than to use a custom syntax like this.